services/core-api/app/config.py

```diff
@@ -86,6 +86,12 @@ class Config(object):
         'root': {
             'level': FLASK_LOGGING_LEVEL,
             'handlers': ['file', 'console']
+        },
+        'loggers': {
+            'werkzeug': {
+                'level': 'CRITICAL',
```

Comment:
above we have this set using an env variable, I think that would be a good idea here, too. If we have a major production bug, then we could turn on more logging to get more info without doing a prod push.

Reply:
Updated.